src/containers/PipelineRuns/PipelineRuns.js

```diff
@@ -75,6 +94,90 @@ export /* istanbul ignore next */ class PipelineRuns extends Component {
     this.setState({ showCreatePipelineRunModal });
   };
 
+  handleChange = event => {
+    const inputValue = event.target.value;
+    this.setState({
+      currentFilterValue: inputValue
+    });
+  };
+
+  handleAddFilter = event => {
+    event.preventDefault();
+    const currentURL = this.props.match.url;
+    const { currentFilterValue = '' } = this.state;
+    const filterValue = currentFilterValue.replace(/\s/g, '');
+    if (!filterValue.match('([a-z0-9A-Z-_./]:[a-z0-9A-Z-_.],?)+')) {
```

Comment:
I'd store this regex up top as a global const so we can easily refer to it later instead of just in this method